aws/lambda/python/s3_cleanup.py

```
@@ -0,0 +1,58 @@
+except Exception as e:
+    print(f"Error setting lifecycle policy: {e}")
+
+print(f"Waiting 10 minutes before deleting bucket: {bucket_name}...")
+time.sleep(600)
+
+try:
+    objects = s3.list_objects_v2(Bucket=bucket_name)
+    if "Contents" in objects:
+        s3.delete_objects(
+            Bucket=bucket_name,
+            Delete={"Objects": [{"Key": obj["Key"]} for obj in objects["Contents"]]}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)